base-notebook/Dockerfile

```diff
@@ -41,8 +41,16 @@ ENV CONDA_DIR=/opt/conda \
 ENV PATH=$CONDA_DIR/bin:$PATH \
     HOME=/home/$NB_USER
 
+# Add a script that we will use to correct permissions after running certain commands
 ADD fix-permissions /usr/local/bin/fix-permissions
-# Create jovyan user with UID=1000 and in the 'users' group
+
+# Enable prompt color in the skeleton .bashrc before creating the default NB_USER
+# and in the default bash.bashrc for the case where the user home directory gets
+# overridden.
+RUN sed -i 's/^#force_color_prompt=yes/force_color_prompt=yes/' /etc/skel/.bashrc && \
+    sed -i 's/^#force_color_prompt=yes/force_color_prompt=yes/' /etc/bash.bashrc
```

Comment:
You are correct. I mistakenly thought the `bash.bashrc` also respected this flag. I've removed the second line and will update the PR description so that it correctly reflects what the PR enables.